src/components/Onboarding.js

```diff
@@ -0,0 +1,37 @@
+import { h } from 'preact'
+import { useState, useEffect } from 'preact/hooks'
+import { route } from 'preact-router'
+import { useSoftkey, useI18n } from 'hooks'
+import { onboarding } from 'utils'
+
+export const Onboarding = () => {
```

Comment:
I would like to see this component simplified. There is a lot of duplication and a lot of calls to i18n() on each render for the same keys.

Reply:
we can also compile the output in `object { image, background, title, description }`, and listen to `currentIndex`, but this brings extra state which doing the same thing.